operators/src/source/ogr_source.rs

```diff
@@ -708,6 +708,7 @@ where
                 FeatureDataType::Text => {
                     let text_option = match field {
                         Ok(FieldValue::IntegerValue(v)) => Some(v.to_string()),
+                        Ok(FieldValue::Integer64Value(v)) => Some(v.to_string()),
```

Comment:
Wouldn't it be possible to also map datetime to string if it is set?
That was one of the problems I guess.

Reply:
it is possible, but if GDAL detects your chemical element name as a date then I think it's better to output nothing (or maybe an error-string) but not a date if we present it as a text column to the user.

Reply:
it is possible, but if GDAL detects your chemical element name as a date then I think it's better to output nothing (or maybe an error-string) but not a date if we present it as a text column to the user.